client/src/modules/employees/registration/employees.js

```diff
@@ -4,11 +4,11 @@ angular.module('bhima.controllers')
 
 EmployeeController.$inject = [
   'EmployeeService', 'ServiceService', 'GradeService', 'FunctionService',
-  'CreditorGroupService', 'DebtorGroupService', 'util', 'NotifyService',
-  'bhConstants', 'ReceiptModal', 'SessionService'
+  'CreditorGroupService', 'util', 'NotifyService',
+  'bhConstants', 'ReceiptModal', 'SessionService',
 ];
 
-function EmployeeController(Employees, Services, Grades, Functions, CreditorGroups, DebtorGroups, util, Notify, bhConstants, Receipts, Session) {
+function EmployeeController(Employees, Services, Grades, Functions, CreditorGroups, util, Notify, bhConstants, Receipts, Session,) {
```

Comment:
... are you sure this works?  It looks like there should be a syntax error - the trailing comma is allowed on arrays, but not functions.